File: GO/class5/ex26_whiteGray.go
/*Desenvolva um programa que imprime na tela “branco” e “cinza”
alternadamente 𝑛 vezes.*/

package main

import "fmt"

func main() {

	var n int

	fmt.Printf("Digite um numero: ")
	fmt.Scanln(&n)

	for i := 1; i <= n; i++ {

		if i%2 != 0 {

			fmt.Println("Branco")
		} else {

			fmt.Println("Cinza")
		}
	}
}

File: C/class5/ex26_whiteGray.c
/*Desenvolva um programa que imprime na tela “branco” e “cinza”
alternadamente 𝑛 vezes.*/

#include <stdio.h>

int main() {

    int n;

    printf("Digite um numero: ");
    scanf("%d", &n);

    for(int i = 1; i <= n; i++) {

        if(i % 2 != 0) {
        printf("Branco\n");
        }
        else {
        printf("Cinza\n");
        }
    }

    return 0;
}